src_py/apiServer/transmitter.py

```diff
@@ -39,8 +40,13 @@ def clientsTraining(self):
     def updateCSV(self, currentPhase): # currentPhase is either "Training", "Prediction" or "Statistics". 
         print('Update CSV Phase')
 
-        #split data and send to mainServer:
-        csvfile = open(globe.INPUT_DATA_PATH+globe.experiment_flow_global.expFlow['CSV path']+"_"+currentPhase.lower()+".csv", 'r').readlines()
+        #split data by sources and send to mainServer:
+        for root, dirnames, filenames in os.walk(globe.INPUT_DATA_PATH):
+            for filename in filenames:
+                if filename == globe.experiment_flow_global.expFlow['CSV path']+"_"+currentPhase.lower()+".csv":
```

Comment:
Instead of concatenating strings with "+" you can use format the following way: 
f"{globe.experiment_flow_global.expFlow['CSV path']}_{currentPhase.lower()}.csv" 
It looks better and easier to read.

Reply:
Done